app/clients/cbc_proxy.py

```diff
@@ -163,6 +160,22 @@ def create_and_send_broadcast(
         }
         self._invoke_lambda(payload=payload)
 
+    def cancel_broadcast(
+        self,
+        identifier, previous_provider_messages,
+        sent, message_number=None
+    ):
+        payload = {
+            'message_type': 'cancel',
+            'identifier': identifier,
+            'message_format': 'cap',
+            "references": [
```

Comment:
Nice 👍 As discussed the reason for the array is because the CAP standard requires all references to a previous alert to be concatenated. E.g.
```
Alert->Update (references Alert)->Cancel (references Update & Alert)